chia/consensus/cost_calculator.py

```diff
@@ -27,7 +27,7 @@ def calculate_cost_of_program(program: SerializedProgram, npc_result: NPCResult,
     # Add cost of conditions
     npc: NPC
     for npc in npc_list:
-        for condition, cvp_list in npc.condition_dict.items():
```

Comment:
@richardkiss @wjblanke I think we'll need to revert this. I hadn't realized that the `conditions_dict` actually collated all the conditions by type. Even though all the types allow for this change, `len(cvp_list)` means something else with this change. It refers to the arguments to the condition, whereas previously it meant the number of conditions of that type.

Reply:
I think a more intuitive way to do this would be to (instead of reverting), removing the `len(cvp_list)` factor, and just add them all up.

Reply:
for now we should just revert this I think